src/api/tests/test_sample_unit_method_obs.py

```diff
@@ -344,6 +345,6 @@ def test_bleaching_quadrat_benthic_percent_field_report(
     assert float(rows[3]["Latitude"]) == site1.location.y
     assert float(rows[3]["Longitude"]) == site1.location.x
     assert rows[3]["Observers"] == profile1.full_name
-    assert rows[3]["Soft coral (% cover)"] == str(
-        obs_quadrat_benthic_percent1_4.percent_soft
+    assert pytest.approx(float(rows[3]["Soft coral (% cover)"]), 1) == pytest.approx(
```

Comment:
Interesting. I'm sure it's right, but curious why `float` and not `Decimal` (and wouldn't that remove the need for the `approx`).